packages/commcare/test/index.js

```diff
@@ -653,3 +653,37 @@ describe('createUser', () => {
     expect(response.statusCode).to.equal(201);
   });
 });
+
+
+describe('HTTP wrappers', () => {
```

Comment:
hmm, since we already have a describe group for `get`, I think we should just call this block `request`